annotation/app/models.py

```diff
@@ -419,3 +420,21 @@ def __repr__(self) -> str:
             f"similar_job_id={self.similar_job_id}, "
             f"label={self.label})>"
         )
+
+
+class AgreementMetrics(Base):
+
+    __tablename__ = "agreement_metrics"
+    task_from = Column(
+        INTEGER,
+        ForeignKey("tasks.id", ondelete="cascade"),
+        primary_key=True,
+        unique=False,
+    )
+    task_to = Column(
+        INTEGER,
+        ForeignKey("tasks.id", ondelete="cascade"),
+        primary_key=True,
+        unique=False,
```

Comment:
May you clarify why we mark these as `primary_key=True` and `unique=False` for both columns? I am a bit unfamiliar with this: what's the purpose of this? What if we get agreement score for same tasks multiple times? It will erase the old one? So we only have only one agreement score at the same time?

Reply:
here is an example of agreement metric for 3 annotators, i believe it expands why do we need `unique=False` for those columns:

task_from, task_to, agreement_score
1, 2, 0.95
1, 3, 0.98
2, 3, 0.97

If i understand business requirements correctly there is no case for getting agreement score for same tasks multiple times - agreement score external service must be called exact once for task group. I believe in potential case of `re-score` metrics we should implement API with UPDATE functionality.

To be honest  i have no clue should we avoid `primary_key=True` for those columns - we need a primary key anyways so we can add column like `id` but i think it will be useless. I guess it's an open question.